index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Quiz Dezvoltare Web</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
+    <link rel="stylesheet" href="/css/style.css">
+    <script defer src="/javascript/quiz.js"></script>
+    <script defer src="/javascript/timer.js"></script>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+</head>
+<body>
+    <div class="card bg-light p-3 border rounded container ceva" >
+        <div>
+            <h2 class="h2-quiz">Quiz moto</h2>
+            <div id="js_timer" class="cont-timer">
+                <div id="timer" class="timer"> 
+                    00:00
+                </div>
+                <div class="timer-start-btn">
+                    <p class="text-uppercase fw-bold" >Start Quiz</p>
+                    <button id="startTimer" class="start-btn"><span class="material-icons">play_arrow</span> </button>
+                </div>
+            </div>
+        </div>
+       
+    <form id="quizForm" class="d-none">
+        <div class="question">
+            <p >
+                1. Care este una dintre cele mai faimoase mărci de motociclete din Statele Unite?
+            </p>
+            <div class="options">
+                <input type="radio" name="q1" value="f" > Honda<br>
```

Comment:
La radio-uri și checkbox-uri, e bine să punem label, pe care userul să poată apăsa pentru a activa. 

Și nu îmi place nici cum e folosit `<br />` aici. Dacă folosești label, cum sugerez eu, ai putea apoi să pui `display: block` pe toate label-urile, sau `display: grid` pe `.options`.

```suggestion
                <label><input type="radio" name="q1" value="f" > Honda </label>
```